klog.go

```diff
@@ -1590,12 +1646,24 @@ func Errorln(args ...interface{}) {
 	logging.println(errorLog, logging.logr, logging.filter, args...)
 }
 
+// ErrorlnDepth acts as Errorln but uses depth to determine which call frame to log.
+// ErrorlnDepth(0, "msg") is the same as Errorln("msg").
+func ErrorlnDepth(depth int, args ...interface{}) {
+	logging.printDepth(errorLog, logging.logr, logging.filter, depth, args...)
```

Comment:
Shouldn't this be `logging.printlnDepth`? `Errorln` uses `logging.println`.

Reply:
hmm, yea